libs/ha/ha_common_libs_k8s.py

```diff
@@ -1143,14 +1148,14 @@ def get_nw_iface_node_down(host_list: list, node_list: list, node_fqdn: str):
                 resp = system_utils.check_ping(host=node_ip)
                 if not resp:
                     return False, node_ip, node_iface, new_worker_obj
-                else:
-                    return True, node_ip, node_iface, new_worker_obj
+                return True, node_ip, node_iface, new_worker_obj
+            return False, "Worker node and Fqdn of data node not same"
```

Comment:
why do we want to return before for loop concluded?

Reply:
Yes this should be after the loop.
Corrected now.